### PR TITLE
Rvv 0.9

### PR DESCRIPTION
**Pull request by issuehsu**
Release candidate that provides:
- GCC 10.1, with RVV enabling options and fp16 supported
- Binutils 2.24, with RVV-0.9 instructions and fp16 supported
- Newlib 3.2, with string and memory functions default optimized for performance
- Gdb 9.1

### CODE REVIEW

**Comment by cgsfv** [comment]
It says:
Newlib 3.2, with string and memory functions default optimized for performance
Gdb 9.1

Have those been updated previously?

File: Makefile [hunk 1]
```diff
@@ -129,16 +129,16 @@ SRC_LIBFTDI  := $(SRCDIR)/libftdi
...
+RGT_VERSION ?= 10.1.0-2020.07.0
+RGDB_VERSION ?= 9.1.0-2020.07.0
+RGBU_VERSION ?= 2.34.0-2020.07.0
+ROCD_VERSION ?= 0.10.0-2020.07.3
+RQEMU_VERSION ?= 4.2.0-2020.07.0
+XC3SP_VERSION ?= 0.1.2-2020.07.0
+TDC_VERSION ?= 0.9.0-2020.07.0
+SDKU_VERSION ?= 0.1.0-2020.07.0
+PY_VERSION ?= 2.7.0-2020.07.0-preview1
+FT_VERSION ?= 2020.07.0
```
Comment (cgsfv) [inline Makefile:142]: Please revert the version numbers for non-toolchain packages.

**Comment by issuehsu** [comment]
> It says:
> Newlib 3.2, with string and memory functions default optimized for performance
> Gdb 9.1
> 
> Have those been updated previously?

Gdb is updated to 9.1 with some patches; previous release is 8.3.
newlib is also updated to 3.2 with some patches; previous release is 3.0.

**Comment by cgsfv** [comment]
> > It says:
> > Newlib 3.2, with string and memory functions default optimized for performance
> > Gdb 9.1
> > Have those been updated previously?
> 
> Gdb is updated to 9.1 with some patches; previous release is 8.3.
> newlib is also updated to 3.2 with some patches; previous release is 3.0.

So that is already in master branch, right?

**Comment by issuehsu** [comment]
> > > It says:
> > > Newlib 3.2, with string and memory functions default optimized for performance
> > > Gdb 9.1
> > > Have those been updated previously?
> > 
> > 
> > Gdb is updated to 9.1 with some patches; previous release is 8.3.
> > newlib is also updated to 3.2 with some patches; previous release is 3.0.
> 
> So that is already in master branch, right?

newlib is already in master branch.
And I found gdb is not updated correctly, Nelson will help to update it soon.

**Comment by issuehsu** [comment]
gdb updated to correct version and features support now.

**Pull request merged**